### DISCUSSION TITLE
There are problems with the latest version of vsg

### DISCUSSION THREAD

**Discussion by WdsPjm** [vsg-dev/VulkanSceneGraph#1013] [Q&A] [answered]
I recently updated vsg, but the shader in vsgexamples doesn't seem to parse properly. It's the same mistake I made when I changed my computer.

![image](https://github.com/vsg-dev/VulkanSceneGraph/assets/52599568/83346161-8504-4d60-9f1d-8b3ae9eb4af1)

**Comment by timoore** [comment]
It looks like your VSG was compiled without the shader compiler, GLSLang.  Try running `git submodule update` in your VSG source tree and compile it again.

**Comment by robertosfield** [comment, marked as answer]
What version of the VSG were you using before you updated?  Are you using VSG master now?

As a general note, the VSG for a several point releases has been pulling in GLSlang as a submodule as compiling it as part of the VSG itself, so if you used a VSG version before this then perhaps your update didn't trigger the VSG to pull this in.  Or perhaps your connection didn't allow it to run correctly.

When GLSlang submodule is correctly checked out you should find the source for it in VulkanSceneGraph/src/glslang.

**Comment by WdsPjm** [comment]
I used to use version 1.0.9. The day before yesterday has built a folder, to clone the VSG on the master branch vsgxchange vsgexamples. Compile all no problem VulkanSceneGraph/SRC/glslang can also find the source code.

**Reply by robertosfield** [reply]
Is it working now?

The CMake var VSG_SUPPORTS_ShaderCompiler should be 1 if everything is checkout correctly.

**Comment by WdsPjm** [comment]
It's working properly
